src/aims_ui/config/dev.py

```diff
@@ -2,6 +2,6 @@
 import os
 
 SECRET_KEY = b'secretkey'
-API_URL = 'http://localhost:9001'
 API_URL = 'https://initial-test-bulk-1.aims.gcp.onsdigital.uk'
```

Comment:
Should point this to: https://analysis-prod-aims-bulk-1.gcp.onsdigital.uk the initail-test one won't exist soon.